src/main/java/com/google/sps/data/Recommended.java

```diff
@@ -0,0 +1,68 @@
+package com.google.sps.data;
+
+import java.util.*;
+
+public class Recommended {
+
+  public static Drink getDrink(String storeid, String drinkName) {
+    List<Drink> drinklist = DrinkDAO.getDrinksByStore(storeid);
+    for (Drink d : drinklist) {
+      if (d.getName().equals(drinkName)) {
+        return d;
+      }
+    }
+    return null;
+  }
+  // given a coffeeshop, return its tydrecommended score
+  public static double getScore(Drink d) {
+    double comments = d.getNumRatings();
+    // instead of shop rating get drink rating from this store
+    double rating = d.getRating();
+    double distanceMinutes = getDistanceMins();
+    double commentScore, ratingScore, distanceScore;
+    //      /30             /35         /35
+
+    // get commentScore
+    if (comments >= 100.0) // 100+
+    commentScore = 30;
+    else if (comments >= 75.0) // 75-99
+    commentScore = 25;
+    else if (comments >= 50.0) // 50-74
+    commentScore = 20;
+    else if (comments >= 25.0) // 25-49
+    commentScore = 15;
+    else commentScore = comments * .6;
+
+    // get ratingScore
+    ratingScore = rating * 7;
+
+    // get distanceScore
+    if (distanceMinutes <= 4.0) // 0-4 mins
+    distanceScore = 35;
+    else distanceScore = 35 - distanceMinutes;
+
+    return (commentScore + ratingScore + distanceScore);
+  }
+  // this function returns the highest rated store from a list of stores
+  public static String getBestShop(List<String> listStoreIds, String inputBeverage) {
```

Comment:
Instead of "listStoreIds", suggest "storeIds". This way, you don't embed the data structure in the name, and can switch it later without having to change the variable name.